libs/sdk-ui-charts/src/highcharts/chartTypes/_chartOptions/chartOptionsBuilder.ts

```diff
@@ -384,7 +386,8 @@ export function getChartOptions(
         : getCategories(type, measureGroup, viewByAttribute, stackByAttribute);
 
     // Pie charts dataPoints are sorted by default by value in descending order
-    if (isOneOfTypes(type, sortedByMeasureTypes)) {
+    // skip this when enableChartSorting is set to true and chart type is PIE | DONUT
+    if ((isPieOrDonutChart(type) && !config.enableChartSorting) || isFunnel(type)) {
```

Comment:
Can we keep using the `isOneOfTypes(type, sortedByMeasureTypes)` or why does Funnel now behave differently?

Reply:
Because Funnel doesn't support custom sort. So it needs this default one independent of FF `enableChartSorting` value.